typescript/sdk/src/middleware/account/InterchainAccount.ts

```diff
@@ -56,4 +74,149 @@ export class InterchainAccount extends RouterApp<InterchainAccountFactories> {
     );
     return new InterchainAccount(helper.contractsMap, helper.multiProvider);
   }
+
+  async deployAccount(
+    chain: ChainName,
+    config: AccountConfig,
+    routerOverride?: Address,
+    ismOverride?: Address,
```

Comment:
I wonder if these should be on the `AccountConfig`?

Reply:
doesn't like come under the "missing artifacts and config" comment?

Reply:
did you make this change or not?